tox.ini

```diff
@@ -1,35 +0,0 @@
-[tox]
```

Comment:
should we delete tox right now?

Reply:
I don't see a reason to continue using tox as GitHub actions can cover everything required.
Less tools, less trouble.

Reply:
You can even run GitHub actions locally in a proper way with https://github.com/nektos/act
This will also use the proper Python versions whereas tox locally uses the installed 3.x release for all 3.x tests as it's based on virtualenv. So no proper 3.6, 3.7, 3.8, etc tests locally with tox.